ios/Classes/SwiftOitiLiveness2dPlugin.swift

```diff
@@ -101,16 +100,16 @@ public class SwiftOitiLiveness2dPlugin: NSObject, FlutterPlugin, FaceCaptchaDele
         let rawEnvironment = args?["environment"] as? String ?? ""
         let environment = Environment(rawValue: rawEnvironment) ?? .HML
         
-        
         let builder = DocumentscopyCustomizationBuilder.builder()
+        let hybridCustomization = HybridDocumentscopyCustomizationBuilder.builder(withNativeBuilder: builder)
```

Comment:
O que você acha dessa sintaxe, faz sentido?

```suggestion
        let builder = DocumentscopyCustomizationBuilder.builder()
        let hybridBuilder = HybridDocumentscopyCustomizationBuilder.builder(withNativeBuilder: nativeBuilder)
        
        var customizationTheme: HybridDocumentscopyCustomizationTheme?
        if custom != nil {
            customizationTheme = createCustomization(
                builder: builder,
                hybridBuilder: hybridBuilder, 
                custom: custom
            ).build()
        }
```
Dessa forma, removeríamos o conteúdo da linha `customizationTheme: (custom != nil) ? createCustomization(builder: builder,hybridBuilder: hybridCustomization, custom: custom).build() : nil` e deixaríamos somente `customizationTheme: customizationTheme` .

Reply:
Feito, só ajustei o `(withNativeBuilder: nativeBuilder)` para `(withNativeBuilder: builder)` pra funcionar